frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseEnhancedGraphTable.java

```diff
@@ -5,16 +5,16 @@
 import org.checkerframework.checker.guieffect.qual.UI;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
 
-public abstract class AbstractEnhancedGraphTable extends BaseGraphTable {
+/**
+ *  {@link BaseEnhancedGraphTable} compared to SimpleGraphTable has graph table refreshing
```

Comment:
Link `SimpleGraphTable`; also, as a general rule, avoid self-linking javadocs (as with `@link BaseEnhancedGraphTable` here)

Reply:
(since they're not obviously very useful)